src/extension.ts

```diff
@@ -504,11 +510,15 @@ function installCustomRequestHandlers(context: VSCode.ExtensionContext) {
   languageClient.onNotification(protocol.ShowIssueOrHotspotNotification.type, showAllLocations);
   languageClient.onNotification(protocol.NeedCompilationDatabaseRequest.type, notifyMissingCompileCommands(context));
   languageClient.onRequest(protocol.GetTokenForServer.type, serverId => getTokenForServer(serverId));
-  languageClient.onNotification(protocol.SubmitTokenNotification.type, token => handleTokenReceivedNotification(token));
   languageClient.onNotification(protocol.PublishHotspotsForFile.type, hotspotsPerFile => {
     hotspotsTreeDataProvider.refresh(hotspotsPerFile);
     updateSonarLintViewContainerBadge();
   });
+  languageClient.onNotification(protocol.AssistCreatingConnection.type, assistCreatingConnection(context));
+  languageClient.onNotification(
+    protocol.AssistBinding.type,
+    async params => await BindingService.instance.assistBinding(params)
```

Comment:
SQ is complaining about this line and I don't understand how to fix it. If I simply put `BindingService.instance.assistBinding`, the method is not being called at all.

Reply:
I marked it as won't fix (probably should have been false positive actually) - there was a discussion with the JS/TS/CSS squad that this rule raised a lot of FP on SQ and SC so they won't enable it by default.